roll/extensions.py

```diff
@@ -84,3 +88,29 @@ def simple_server(app, port=3579, host='127.0.0.1', quiet=False):
         app.loop.run_until_complete(app.shutdown())
         server.close()
         app.loop.close()
+
+
+def static(app, prefix='/static/', root=Path()):
+    """Serve static files. Never use in production."""
+
+    root = Path(root).resolve()
+
+    if not prefix.endswith('/'):
+        prefix += '/'
+    prefix += '{path:path}'
+
+    async def serve(request, response, path):
```

Comment:
I wonder if this function should exist outside of `static`, so it can be used as a util (something like `serve_from_file`).
I've also been tempted to make a shortcut out of it, something like:

    response.file = Path('path/to/file')

Which would do what `serve` currently does (raise if file does not exist, guess type, set header, etc.).
But I feel it's a bit too much job done to be put onto a shortcut.

Thoughts?

Reply:
I think using `response.file` directly would encourage to keep that in production.

What would be the correct way to transition to serving file from nginx? Maybe a good thing to add to the documentation?

Reply:
> I think using response.file directly would encourage to keep that in production.

Agreed.

> What would be the correct way to transition to serving file from nginx? 

Just a directive that comes before Roll.

> Maybe a good thing to add to the documentation?

Yes, I think the "deploy to production" part of the documentation needs more love. But I think it's outside of this given PR.